### ISSUE TITLE
CDR: Get job failure rate and correlation in failure rate from condor job monitoring

### ISSUE THREAD

**Issue by khurtado** [dmwm/WMCore#11550]
**Impact of the new feature**
WM in general

Is your feature request related to a problem? Please describe.
This issue is part of #11408, in order to collect information of job failure rate and correlation in failure rate in the system.

**Describe the solution you'd like**
Using Condor Job Monitoring, plot for the last year:
Classification of job exit code vs architecture (TODO: which classads to use?)
Classification of WMAgent number of retries
Correlation/distribution of NumJobStarts
CMS Monit ticket: https://its.cern.ch/jira/browse/CMSMONIT-533

**Describe alternatives you've considered**
None

**Additional context**
None

**Comment by khurtado** [comment]
Follow up on:

https://its.cern.ch/jira/browse/CMSMONIT-533

**Comment by amaltaro** [comment]
As can be seen in the ticket above, a grafana dashboard named "Production jobs exit code monitoring" has been provided and it's linked in the CMS Monitoring frontend page (under WMAgent).

I have also analyzed some of that data and I updated the gdoc "Evaluation of the WM system for the HL-LHC scenario", under the "Current WM system" section.

I am closing this ticket out, but feedback is appreciated in the google document.
